pkgs/applications/networking/cluster/openshift/default.nix

```diff
@@ -14,21 +14,32 @@ in buildGoPackage rec {
     owner = "openshift";
     repo = "origin";
     rev = "v${version}";
-    sha256 = "0qvyxcyca3888nkgvyvqcmybm95ncwxb3zvrzbg2gz8kx6g6350v";
+    sha256 = "08bdqvsjl6c7dmllyz8n4akb7gyn91znvbph5cgmmk1bhskycy1r";
   };
 
   buildInputs = [ which ];
 
   goPackagePath = null;
   patchPhase = ''
     patchShebangs ./hack
+    substituteInPlace pkg/bootstrap/docker/host/host.go  \
+      --replace 'nsenter --mount=/rootfs/proc/1/ns/mnt findmnt' \
+      'nsenter --mount=/rootfs/proc/1/ns/mnt ${utillinux}/bin/findmnt'
```

Comment:
Please also make replace `nsenter` with `${utillinux}/bin/nsenter`

Reply:
That will break stuff! These `nsenter`'s are run from within the openshift container. It enters the namespace of the host of the container, and then executes e.g. findmt, mount & mkdir in there.